### ISSUE TITLE
Long inline code will be displayed without line breaking and ends up outside of viewport

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/t3SphinxThemeRtd#138]
I am not sure what we should do: probably recommend to not use inline code for long code fragments or handle the rendering differently. 

-----

https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87583-RemoveObsoleteAPCCacheBackendImplementation.html

![changelog](https://user-images.githubusercontent.com/13206455/68370205-3f4e8f00-013c-11ea-9c56-8ec02d70b72b.png)

Without any styling, the existing code would break:

```
<html>

<code class="code php docutils literal notranslate">
<span class="pre">$GLOBALS['TYPO3_CONF_VARS']['SYS']['caching']['cacheConfigurations']['rootline']['backend']</span> 
<span class="pre">=</span> 
<span class="pre">\TYPO3\CMS\Core\Cache\Backend\ApcBackend::class;</span>
</code>

</html>
```

**Comment by sypets** [comment]
My opinion: Often, the use of inline code is not a good choice if the text is too long. See also https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/723

The theme now fixes this (#140), but we should probably also have a long hard look at the places where this does occur and evaluate if the used formatting is a good choice for readability.

**Comment by benjaminkott** [comment]
Resolved with https://github.com/TYPO3-Documentation/t3SphinxThemeRtd/pull/140
